test/test_red_black_tree.jl

```diff
@@ -0,0 +1,77 @@
+include("../src/red_black_tree.jl")
+@testset "RBTree" begin
+    t = RBTree{Int}()
+    for i = 1:10000
```

Comment:
The rest of the package fairky  consistently uses
```suggestion
    for i in 1:10000
```